src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -119,7 +128,10 @@ class ActionsPanel extends React.PureComponent<Props> {
 
               return (
                 <PanelItemGrid key={i}>
-                  {ActionLabel[action.type]}
+                  {this.getFullActionTitle({
+                    type: action.type,
+                    integrationName: availableAction && availableAction.integrationName,
```

Comment:
@billyvg optional chaining is now merged but I will approve anwyway

Reply:
I can follow up with a PR to change this

Reply:
@billyvg up to you